dataactbroker/handlers/fileHandler.py

```diff
@@ -831,6 +832,9 @@ def submit_detached_file(submission):
             for row in query:
                 # remove all keys in the row that are not in the intermediate table
                 temp_obj = row.__dict__
+
+                temp_obj['business_categories'] = get_business_categories(row=temp_obj, data_type='fabs')
```

Comment:
This should go into the derivations, not outside them. This is one of the derivations.

Related: This should have tests